workers/unity/Packages/io.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -209,28 +199,14 @@ private static string GetClientLogFileFullPath()
             }
         }
 
-        public static void ConfigureLocalDeployment()
+        public static void LaunchLocalDeployment()
```

Comment:
This should do a safety check to see if the snapshot file exists and print an error.

Reply:
This is already done on the launch execution and adds an error message on the log file (as well as an error on the Console)

On the Log file

`error Encountered an error during command execution. [stack: {code = NotFound desc = Could not find initial snapshot file 'C:\Users\jaysantos\Documents\GDK\gdk-for-unity\custom 32.snapshot'}][original_log_level: fatal][cli-version: 20190724.082439.ce7ba70f56][error: code = NotFound desc = Could not find initial snapshot file 'C:\Users\jaysantos\Documents\GDK\gdk-for-unity\custom 32.snapshot'] [timestamp: 2019-08-07T16:50:42+01:00]`

On the console:

`Errors occured - For more information, check the spatial local launch logfile: C:\Users\jaysantos\Documents\GDK\gdk-for-unity\logs\spatial_20190807T165041.log
UnityEngine.Debug:LogError(Object)
Improbable.Gdk.Tools.<>c__DisplayClass14_0:<LaunchLocalDeployment>b__0(Object, EventArgs) (at Packages/io.improbable.gdk.tools/LocalLaunch.cs:274)
System.Threading._ThreadPoolWaitCallback:PerformWaitCallback()`

Reply:
True but this is an unreadable error by toolbelt. A quick check if the file exists would be much nicer and avoids users running into these.